geniza/corpus/models.py

```diff
@@ -264,6 +265,14 @@ def is_public(self):
     is_public.boolean = True
     is_public.admin_order_field = 'status'
 
+    def get_absolute_url(self):
+        return reverse('corpus:document', args=[str(self.id)])
+
+    def iiif_urls(self):
+        """List of IIIF urls for images of the Document's Fragments."""
+        return list(filter(None, self.fragments.values_list("iiif_url",
```

Comment:
If we access fragments via `text_block` it should honor order (if any is specified). Might not matter for now, not sure.

Reply:
oh good call, thx for the reminder